src/games.js

```diff
@@ -0,0 +1,25 @@
+import renderTable from './renderTable.js';
+
+const FINAL_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/iLhUHUKbK0GFKQnhs8n8/scores/';
+
+const apiRefreshScores = async () => {
+  await fetch(`${FINAL_URL}`)
+    .then((response) => response.json())
+    .then((json) => renderTable(json));
+};
```

Comment:
- [ ] According to the project requirement, use async and await your promise, your code can be cleaner without `then` method. you can change it to something like this :point_down: 
```
const apiRefreshScores = async () => {
 let data = await fetch(`${FINAL_URL}`)
 let resp = await data.json()
 renderTable(resp)
};
```